app/thirdparty/java/other/writeily/widget/WrMarkorWidgetProvider.java

```diff
@@ -57,6 +61,7 @@ public void onUpdate(Context context, AppWidgetManager appWidgetManager, int[] a
                     .setAction(Intent.ACTION_SEND)
                     .putExtra(Intent.EXTRA_TEXT, "");
             views.setOnClickPendingIntent(R.id.widget_new_note, PendingIntent.getActivity(context, requestCode++, openShare, staticFlags));
+            views.setInt(R.id.widget_new_note, "setColorFilter", color);
```

Comment:
Setting icon tint in XML does not work for some reason. Without this we have some black and some white icons